pages/index.tsx

```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
         welcomeMessage="selamat datang"
         title="Komunitas Developer ReactJS Indonesia"
       />
-      <PageBody>
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Certe non potest. Negabat igitur ullam esse artem, quae ipsa a se
-          proficisceretur; Duo Reges: constructio interrete. Quid turpius quam sapientis vitam ex insipientium sermone pendere? Non igitur
-          bene. Vide, quaeso, rectumne sit.
-        </p>
-        <p>
-          Last updated: <time dateTime={lastUpdated}>{relativeTime(new Date(lastUpdated))}</time> |{' '}
-          <Link href="/about" passHref>
-            <a>About</a>
-          </Link>
-        </p>
-      </PageBody>
+
+      <Box backgroundColor="#f2f2f2">
+        <Grid gridTemplateColumns="max-content 1fr" columnGap="5em" alignItems="center" py="6em" px="11%">
+          <Image display={['hidden', null, null, '']} size="400px" src="/rumah-komunitas/tshirt.jpg" alt="tshirt from rumah komunitas" />
+          <Grid>
+            <Heading as="h2" fontSize="xl" fontWeight={300} textTransform="uppercase">
+              kabar gembira!
+            </Heading>
+            <Heading as="h3" fontSize="2.86rem" fontWeight={600}>
+              Merchandise Resmi ReactJS Indonesia
+            </Heading>
+            <Text as="p" my="18px">
+              Merchandise resmi ReactJS Indonesia kini tersedia berkat kerjasama oleh Rumah Komunitas. Klik link di bawah untuk mendapatkan
+              T-shirt dan jaket ReactJS Indonesia.
+            </Text>
+            <LinkButton ml="-16px" href="https://www.rumahkomunitas.com/react-indonesia" isExternal>
+              Dapatkan Segera
+            </LinkButton>
+          </Grid>
+        </Grid>
+      </Box>
+
+      <Box backgroundColor="lightBlue" paddingX="8.5em" paddingY="6em">
+        <Grid gap="2px" justifyContent="center" gridAutoFlow="row" textAlign="center">
+          <Heading as="h2" color="gray08" fontSize="20px" lineHeight="28px" fontWeight={300} textTransform="uppercase">
+            ingin belajar react?
+          </Heading>
+          <Heading as="h3" fontSize="48px" lineHeight="56px">
```

Comment:
```suggestion
          <Heading as="h3" fontSize="md" fontWeight={600}>
```

Font weight should be 600. Proper type scale.